spec/lib/govuk_publishing_components/presenters/related_navigation_helper_spec.rb

```diff
@@ -212,7 +203,6 @@ def payload_for(schema, content_item, context = nil)
       expected = [
         { locale: "en", text: "Travel abroad", path: "/browse/abroad/travel-abroad" },
         { locale: "en", text: "Arriving in the UK", path: "/browse/visas-immigration/arriving-in-the-uk" },
-        { locale: "en", text: "Pets", path: "/topic/animal-welfare/pets" },
```

Comment:
Is this line intentional? Isn't it a specialist topic?

Reply:
Not sure I fully understand your question... When content is tagged to browse and also to a taxonomy topic, we show the browse page in the footer. So in this test, I've replaced the specialist topic in the links with a taxonomy topic. And updated the expected out put to highlight that only the browse links are displayed.

Reply:
Oh, wait, my mistake.